src/main/java/net/mcreator/ui/minecraft/boundingboxes/JBoundingBoxList.java

```diff
@@ -72,13 +95,48 @@ public JBoundingBoxList(MCreator mcreator, IHelpContext gui) {
 		boundingBoxList.forEach(e -> e.setEntryEnabled(enabled));
 	}
 
+	public void modelChanged() {
+		if (modelProvider != null)
+			genFromModel.setVisible(modelProvider.get() != null && modelProvider.get().getType() == Model.Type.JSON);
+	}
+
+	private void generateBoundingBoxFromModel() {
+		if (modelProvider != null) {

```

Comment:
It may be a good idea to wrap this in a try-catch and show some sort of warning it model extraction fails.

One can never be sure what all sorts of kinds of things users will import as a JSON model hehe

Reply:
Should be errors logged or re-thrown?

Reply:
Logged + user dialog about how generation of bbox failed

Reply:
You mean showing what part is corrupt in that dialog?

Reply:
I would just say "Failed to generated bounding boxes for the selected JSON model" and maybe another line "Make sure your model is a valid Minecraft JSON model"

Reply:
OKay, will do that right now.
(Me who have just modified code to log proper part name)